### PR TITLE
fix: i18n missing and plural items

### PR DESCRIPTION
**Pull request by yomybaby**
### TL;DR

Update i18n translations for multiple languages, adding new translations and removing unused ones.

### What changed?

- Added new translations for various strings in different language files
- Removed unused translations such as 'PreOpenPortMaxCountLimit_plural'
- turn off the plural feature of i18n scanner

### How to test?

No specific instructions for testing provided

### Why make this change?

Improve user experience by providing accurate translations and removing unused translations

---

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by yomybaby** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2414?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2414** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2414?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2405** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2405?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yomybaby and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: resources/i18n/pt.json [hunk 1]
```diff
@@ -1563,14 +1563,11 @@
     "CloneAsFolder": "Clonar como Pasta",
     "CloneSuccess": "O pedido de clone foi enviado com sucesso.",
     "category": "Categoria",
-    "CloneInfo": "__NOT_TRANSLATED__"
+    "CloneInfo": "Ele será clonado como sua pasta de tipo de usuário."
   },
```
Comment (lizable) [inline resources/i18n/pt.json:1567]: @yomybaby 
I found some parts that hasn't been translated yet, so I replaced with auto-translation.

**Review by ironAiken2 — APPROVED** [review]
LGTM

**Comment by lizable** [comment]
### Merge activity

* **May 16, 3:19 AM EDT**: @lizable merged this pull request with [Graphite](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2414).

**Pull request merged**